test/services/db-export/export-data-files.service.test.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const fs = require('fs')
+const path = require('path')
+
+// Thing under test
+const ExportDataFilesService = require('../../../app/services/db-export/export-data-files.service')
+
+const csvHeader = [
+  '"billingChargeCategoryId"',
+  '"reference"',
+  '"subsistenceCharge"',
+  '"description"',
+  '"shortDescription"',
+  '"isTidal"',
+  '"lossFactor"',
+  '"modelTier"',
+  '"isRestrictedSource"',
+  '"minVolume"',
+  '"maxVolume"',
+  '"dateCreated"',
+  '"dateUpdated"'
+]
+
+const csvValues = [
+  '"20146cdc-9b40-4769-aa78-b51c17080d56"',
+  '"4.1.1"',
+  '"9700"',
+  '"Low loss tidal abstraction of water up to and including 25,002 megalitres a year where no model applies"',
+  '"Low loss, tidal, up to and including 25,002 ML/yr"',
+  '"2022-12-14T18:39:45.000Z"',
+  '"2022-12-14T18:39:45.000Z"',
+  '"true"',
+  '"low"',
+  '"no model"',
+  '"false"',
+  '"0"',
+  '"25002"'
+]
+let filePath
```

Comment:
We could move this down into the `when successful` block since this is the only context we use it in.

Reply:
Done :)